frontend/src/app/modules/album/components/create-edit-album/create-edit-album/create-edit-album.component.ts

```diff
@@ -223,4 +229,33 @@ export class CreateEditAlbumComponent implements OnInit, OnDestroy {
         });
     }
   };
+
+  setPublicStatus() {
+    let albumPublicStatus: AlbumPublicStatus;
+
+    if (this.album.isPublished) {
+      albumPublicStatus = {
+        ...this.album,
+        isPublished: false
+      };
+    }
+    else {
+      albumPublicStatus = {
+        ...this.album,
+        isPublished: true
+      };
+    }
+
```

Comment:
Could change it to 
const albumPublicStatus = {
        ...this.album,
        isPublished: !this.album.isPublished
      };